devices/SPS30/Utils/BigEndianBitConverter.cs

```diff
@@ -0,0 +1,48 @@
+﻿//
+// Copyright (c) 2017 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+
+namespace Iot.Device.SPS30.Utils
+{
+    /// <summary>
+    /// The SPS30 does everything big-endian. We need a little helper class to help us with conversion in case BitConverter is little endian on the current platform.
+    /// </summary>
+    public static class BigEndianBitConverter
```

Comment:
Note: you'll miss the params and return types everywhere

Reply:
I was unaware of its existence. However, it currently does not support Float/Single values. I'll leave my BigEndianBitConverter for now. When BinaryPrimitives support Float, this can be refactored.

Reply:
Correct, it's not there *yet* for float (or double). Float are "Half" in the .NET 6.0 implementation for the BinaryPrimitives: https://docs.microsoft.com/en-us/dotnet/api/system.buffers.binary.binaryprimitives?view=net-6.0
Can you at least use the BinaryPrimitives for the UInt16?

Reply:
For this implementation, we'll need the "Single" (32-bit float) instead of "Half" (16-bit float). I've changed the code to use BinaryPrimitives where possible and removed the `BigEndianBitConverter` util class completely. The only remaining method `ReadSingleBitEndian` is only used when parsing measurement and has thus been moved to the `Measurement` class as a private method.